src/services/fcm.service.ts

```diff
@@ -28,22 +29,40 @@ export class FcmService {
       });
     }
 
-    const options = {
-      priority: 'high',
-      timeToLive: 60 * 60 * 24,
-    };
-
-    const optionsSilent = {
-      priority: 'high',
-      timeToLive: 60 * 60 * 24,
-      content_available: true,
-    };
+    const body: firebaseAdmin.messaging.MulticastMessage = {
+      tokens: deviceIds,
+      data: payload?.data,
+      notification: {
+        title: payload?.notification?.title,
+        body: payload?.notification?.body,
+        imageUrl
+      },
+      apns: {
+        payload: {
+          aps: {
+            sound: payload?.notification?.sound,
+            contentAvailable: silent ? true : false,
+            mutableContent: true
+          }
+        },
+        fcmOptions: {
+         imageUrl
+        }
+      },
+      android: {
+        priority: 'high',
+        ttl: 60 * 60 * 24,
+        notification: {
+          sound: payload?.notification?.sound
+        }
+      }
+    }
 
     let result = null;
     try {
       result = await firebaseAdmin
```

Comment:
please update the implementation to send notifications in batch of max 500 device ids. Google currently support now only list of max 500 device ids, ref here: https://firebase.google.com/docs/cloud-messaging/send-message#send-messages-to-multiple-devices.